generators/server/templates/src/main/resources/config/application.yml.ejs

```diff
@@ -125,6 +125,13 @@ management:
         git:
             mode: full
     health:
+        group:
+            liveness:
+                include: livenessState
+            readiness:
+                include: readinessState<% if (databaseType === 'sql') { %>,datasource<% } %>
+        probes:
+            enabled: true
```

Comment:
Do we need when developing locally too, or can it be disabled?

Reply:
I think it doesn't impact startup time significantly... So let's have it all the time.